From 350410fc1d38c4ac8422816b6865ac8cd9c60fc7 Mon Sep 17 00:00:00 2001
From: Alex Gaynor <alex.gaynor@gmail.com>
Date: Thu, 5 Sep 2024 16:24:32 -0400
Subject: [PATCH] Ignore warning that CSRs are deprecated in pyOpenSSL (#185)

Without this, pyca/cryptography's downstream tests currently fail.

The actual warning is being addressed in #182
---
 pyproject.toml | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 4e2009fb..2af02b11 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -96,7 +96,11 @@ disallow_untyped_defs = true
 
 [tool.pytest.ini_options]
 # We also ignore our own deprecation warning about dropping Python 3.7 support.
-filterwarnings = ["error", "ignore:Python 3.7 support will be dropped:DeprecationWarning"]
+filterwarnings = [
+    "error",
+    "ignore:Python 3.7 support will be dropped:DeprecationWarning",
+    "ignore:CSR support in pyOpenSSL is deprecated:DeprecationWarning",
+]
 norecursedirs = "*.egg .eggs dist build docs .tox"
 
 # Isort tooling configuration
